javacutil/src/main/java/org/checkerframework/javacutil/AnnotationUtils.java

```diff
@@ -334,26 +334,52 @@ public static boolean containsSameByName(
         return null;
     }
 
-    private static final Comparator<AnnotationMirror> ANNOTATION_ORDERING =
-            new Comparator<AnnotationMirror>() {
-                @Override
-                public int compare(AnnotationMirror a1, AnnotationMirror a2) {
-                    // AnnotationMirror.toString() prints the elements of an annotation in the
-                    // order in which they were written. So, use areSame to check for equality.
-                    if (AnnotationUtils.areSame(a1, a2)) {
-                        return 0;
-                    }
-
-                    String n1 = a1.toString();
-                    String n2 = a2.toString();
-
-                    // Because the AnnotationMirror.toString prints the annotation as it appears
-                    // in source code, the order in which annotations of the same class are
-                    // sorted may be confusing.  For example, it might order
-                    // @IntRange(from=1, to=MAX) before @IntRange(to=MAX,from=0).
-                    return n1.compareTo(n2);
-                }
-            };
+    /**
+     * Provide ordering for {@link AnnotationMirror}s. AnnotationMirrors are first compared by their
+     * fully qualified names, then by their element values, in ordered of the name of the element.
```

Comment:
Should it be `fully-qualified names`?
Should the `ordered` be `order` or is something else strange?